server/src/tree-sitter/analyzer.ts

```diff
@@ -237,8 +237,9 @@ export default class Analyzer {
     TreeSitterUtils.forEach(bashTree.rootNode, (node) => {
       const isFunctionDefinition = node.type === 'function_definition'
       const isVariableName = node.type === 'variable_name'
+      const isCommandName = node.type === 'command_name' // Note a command is just a function when being called.
```

Comment:
This is tree-sitter-bash terminology, this is the node type given by it. Since we're analyzing the bash tree, I think it is less confusing to keep the same terminology.

Reply:
Absolutely then